test/aria/widgets/container/dialog/modalMaskSize/DialogModalMaskSize.tpl

```diff
@@ -0,0 +1,176 @@
+                resIndex: -1
+            }
+        } /}
+        {@aria:Dialog {
+            modal: true,
+            movable: true,
+            maximizable: true,
+            resizable: true,
+            macro: "dialogContent",
+            bind: {
+                xpos: {
+                    inside: data,
+                    to: "x"
+                },
+                ypos: {
+                    inside: data,
+                    to: "y"
+                },
+                width: {
+                    to: "width",
+                    inside: data
+                },
+                height: {
+                    to: "height",
+                    inside: data
+                },
+                center: {
+                    to: "center",
+                    inside: data
+                },
+                visible: {
+                    to: "visible",
+                    inside: data
+                }
+            }
+        }/}
+        {@aria:CheckBox {
+            label: "Dialog visible",
+            bind: {
+                value: {
+                    to: "visible",
+                    inside: data
+                }
+            }
+        } /}
+        <div style="width: 100px; height: 2000px; background-color:blue;"></div>
+        <input {id "myInput"/}>
+        {@aria:CheckBox {
+            label: "Dialog visible",
+            bind: {
+                value: {
+                    to: "visible",
+                    inside: data
+                }
+            }
+        } /}
+    {/macro}
+
+    {macro dialogContent()}
+        <input><br>
+        {@aria:CheckBox {
+            label: "Center dialog",
+            bind: {
+                value: {
+                    to: "center",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
+            label: "x",
+            bind: {
+                value: {
+                    to: "x",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
+            label: "y",
+            bind: {
+                value: {
+                    to: "y",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
+            label: "width",
+            bind: {
+                value: {
+                    to: "width",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
+            label: "height",
+            bind: {
+                value: {
+                    to: "height",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
+            label: "boxWidth",
+            bind: {
+                value: {
+                    to: "boxWidth",
+                    inside: data
+                }
+            }
+        } /}<br>
+        {@aria:NumberField {
```

Comment:
Are all those fields above useful for manual testing? Otherwise (and only otherwise) I would say to remove them since they are not used in the automated test.

Reply:
Yes, those fields are useful for manual testing.